src/core/utils/urlutils.js

```diff
@@ -141,24 +141,3 @@ export function filterParamsForExperienceLink (
   newParams.delete(StorageKeys.SEARCH_OFFSET);
   return newParams;
 }
-
-/**
- * returns if two SearchParams objects have the same key,value entries
- * @param {SearchParams} params1
- * @param {SearchParams} params2
- * @return {boolean} true if params1 and params2 have the same key,value entries, false otherwise
- */
-export function equivalentParams (params1, params2) {
```

Comment:
this was also on L73! https://github.com/yext/answers/pull/1024/files#diff-cb456038831900f9a9105a30eec6b1f5L73